src/view/com/composer/photos/OpenCameraBtn.tsx

```diff
@@ -43,7 +44,7 @@ export function OpenCameraBtn({gallery}: Props) {
     }
   }, [gallery, track, store, requestCameraAccessIfNeeded])
 
-  if (isDesktopWeb) {
+  if (isDesktop) {
```

Comment:
This was originally intended as a platform test not a device-size test. The interesting question is whether the camera module will work with web.

Reply:
I've revised to use `isNative` and `isMobileWeb` checks for showing the camera button, which should cover native apps and mobile web